angular.json

```diff
@@ -62,7 +62,7 @@
                 {
                   "type": "initial",
                   "maximumWarning": "2.5mb",
-                  "maximumError": "6mb"
+                  "maximumError": "7mb"
```

Comment:
really ? in one month we grow bundle size 1mb.... It's maybe time to rethink our application architecture.

Reply:
Yes, it's time to refactor and put in `lazy loading`.